sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -997,21 +997,21 @@ def __repr__(self):
 
 class MinedOpinion(DictMixin):
     """A mined opinion object represents an opinion we've extracted from a sentence.
-    It consists of both an aspect that these opinions are about, and the actual
-    opinions themselves.
+    It consists of both a target that these opinions are about, and the assessments
+    representing the opinion.
 
-    :ivar aspect: The aspect of a product/service that this opinion is about
-    :vartype aspect: ~azure.ai.textanalytics.AspectSentiment
-    :ivar opinions: The actual opinions of the aspect
-    :vartype opinions: list[~azure.ai.textanalytics.OpinionSentiment]
+    :ivar target: The target aspect of a product/service that this opinion is about
```

Comment:
nit: get rid of aspect / opinion I think, i.e. just "The target of an opinion about. a product / service"

Reply:
Ok, I updated this in a recent commit.  I also modified or removed a lot of other references I noticed that I missed as well.